service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionsService.java

```diff
@@ -47,6 +47,7 @@ public boolean hasEmptyDates(CaseData caseData) {
                 caseData.getCafcassDirections(),
                 caseData.getOtherPartiesDirections(),
                 caseData.getCourtDirections())
+            .filter(Objects::nonNull)
```

Comment:
As discussed I don't think this is the correct solution, the issue should be caused else where as it should not be possible to get here without having these directions, which would be populated in the notify gatekeeper event

Reply:
As discussed we have worked out that the issue is some bad test data. Please update that test data and then everything should be fine.

Reply:
changed based on discussion